File: java/mix/C01FibType1B.java
public class C01FibType1B {
    public static long fib(int n) {
        long a = 0, b = 1;
        for (int i = 0; i < n; i++) {
            long next = a + b;
            a = b;
            b = next;
        }
        return a;
    }
}

File: javascript/mix/C01FibType1B.js
function fib(n) {
    let a = 0, b = 1;

    for (let i = 0; i < n; i++) {
        const next = a + b;
        a = b;
        b = next;
    }
    return a;
}

console.log(fib(10));

module.exports = { fib };
